src/mesh/RadioInterface.cpp

```diff
@@ -123,6 +123,20 @@ const RegionInfo regions[] = {
     */
     RDEF(MY_919, 919.0f, 924.0f, 100, 0, 27, true, true, false),
 
+    /*
+        Singapore
+        SG_868 Band 29: 868 - 869 MHz at 500mW, no restrictions.
+        https://www.imda.gov.sg/-/media/imda/files/regulation-licensing-and-consultations/ict-standards/telecommunication-standards/radio-comms/imdatssrd.pdf
+    */
+    RDEF(SG_868, 868.0f, 869.0f, 100, 0, 27, true, false, false),
+    
+    /*
+        Singapore
+        SG_923 Band 30d: 917 - 925 MHz at 100mW, no restrictions.
+        https://www.imda.gov.sg/-/media/imda/files/regulation-licensing-and-consultations/ict-standards/telecommunication-standards/radio-comms/imdatssrd.pdf
+    */
+    RDEF(SG_923, 917.0f, 925.0f, 100, 0, 20, true, false, false),
```

Comment:
The duty cycle must be reduced from 100 to 1, in keeping with table entry 30c

Reply:
@code8buster see previous comments https://github.com/meshtastic/firmware/pull/3165#discussion_r1475907614